docs/usage.md

```diff
@@ -96,14 +96,14 @@ The `--serve` flag is a helper flag to set the output format to HTML, and serve
 osv-scanner scan -L package-lock.json --serve
 ```
 
-### Experimental features
+### Other features
 
-Several experimental features are available through flags. See their respective documentation pages for more details:
+Several other features are available through flags. See their respective documentation pages for more details:
 
-- [`--experimental-offline-vulnerabilities`](./offline-mode.md)
-- [`--experimental-licenses`](./license-scanning.md)
+- [`--offline-vulnerabilities`](./offline-mode.md)
+- [`--licenses`](./license-scanning.md)
```

Comment:
Now that these are out of experimental, we can have a small section for each of these like the sections above. Just one line and a short example should be good enough here, and then link to the actual page. (e.g. Similar to the format section or config section)